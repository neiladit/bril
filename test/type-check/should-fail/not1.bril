main{

    v1: int = const 1;
    v2: bool = not v1;
    print v2;
}
