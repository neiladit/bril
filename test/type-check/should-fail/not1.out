line 3: assign non-bool values to bool var
logic op error!
